ta1/output.py

```diff
@@ -293,5 +301,40 @@ class Map(BaseModel):
     features: MapFeatureExtractions
     cross_sections: Optional[list[CrossSection]]
 
-    pipelines: list[ModelRun]
+    
     projection_info: ProjectionMeta
+
+
+
+graph = erd.create(Map)
```

Comment:
I think this diagram creation code is now duplicative of code in https://github.com/DARPA-CRITICALMAAS/schemas/blob/main/scripts/generate-schema.py. This code  (and the imports that it references) should be deleted for a clean merge.